src/Agent.Worker/Handlers/NodeHandler.cs

```diff
@@ -115,6 +115,11 @@ public async Task RunAsync()
                 file = GetNodeLocation();
             }
 
+            ExecutionContext.Debug("Using node path: " + file);
+            if (!File.Exists(file)) {
+                throw new FileNotFoundException($"This step requires a node version that does not exist in the agent filesystem. Path: {file}");
```

Comment:
This string should probably get localized

Reply:
Good point. I'll add it